Sources/GDSCommon/UI/UIKit/SharedViews/Components/BulletView.swift

```diff
@@ -22,7 +29,9 @@ public final class BulletView: NibView {
     /// Convenience initaliser to initialise a `BulletView` directly from a ``BulletViewModel``
     /// - Parameter viewModel: ``BulletViewModel``
     public convenience init(viewModel: BulletViewModel) {
-        self.init(title: viewModel.title, text: viewModel.text)
+        self.init(title: viewModel.title,
+                  titleFont: viewModel.titleFont ?? .init(style: .title3, weight: .semibold),
```

Comment:
Not a deal breaker but would it be worth moving and making it the last param as it has a default value?

Reply:
Can do, although I did think that but then paired it with title as it relates to that 🤣 happy to change if we want though, im not fussy

Reply:
Changed now :)